post/urls.py

```diff
@@ -5,22 +5,18 @@
 
 urlpatterns = [
     # /post/
-    #url(r'^$', views.index, name='index'),
     url(r'^$', views.IndexView.as_view(), name='index'),
 
     # /post/71/
-    #url(r'^(?P<post_id>[0-9]+)/$', views.detail, name='detail'),
     url(r'^(?P<pk>[0-9]+)/$', views.DetailView.as_view(), name='detail'),
 
     # /post/add/
     url(r'add/$', views.PostCreate.as_view(), name='post-add'),
 
-
-    #This is to update post. Currently not set-up in the front-end
+    # This is to update post. Currently not set-up in the front-end
     # /post/71/
     url(r'(?P<pk>[0-9]+)/$', views.PostUpdate.as_view(), name='post-update'),
 
-
     # /post/71/delete/
     url(r'(?P<pk>[0-9]+)/delete/$', views.PostDelete.as_view(), name='post-delete'),
 ]
```

Comment:
A minor style note by today's standard but POSIX (and by extension, PEP8) dictates that files should end in a newline. For your reading pleasure: http://stackoverflow.com/a/729795/2557554